src/main/java/uk/gov/hmcts/ccd/domain/service/validate/DefaultValidateCaseFieldsOperation.java

```diff
@@ -62,15 +69,15 @@ private void validateOrganisationPolicy(String caseTypeId, CaseDataContent conte
         caseDefinitionRepository.getCaseType(caseTypeId)
             .findCaseEvent(content.getEventId())
             .ifPresent(caseEventDefinition -> caseEventDefinition.getCaseFields()
-                .stream()
                 .forEach(eventFieldDefinition -> eventFieldDefinition.getCaseEventFieldComplexDefinitions().stream()
                     .filter(cefcDefinition -> isOrgPolicyCaseAssignedRole(cefcDefinition.getReference()))
                     .forEach(cefcDefinition -> {
                         String reference = cefcDefinition.getReference();
-                        validateContent(content, eventFieldDefinition.getCaseFieldId(),
-                            reference,
-                            cefcDefinition.getDefaultValue(),
-                            errorList);
+                        validateContent(content,
```

Comment:
Probably nothing to do with your PR @rafalkalita, but I just noticed that we are adding custom validation rules (for the organisation) in this class. But this class is a generic one! Why is a method like:
`private void validateOrganisationPolicy(String caseTypeId, CaseDataContent content) {` 
here?